io-backend/src/main/java/agh/io/iobackend/model/GameRoom.java

```diff
@@ -20,35 +23,70 @@ public class GameRoom {
     )
     private Long gameRoomID;
 
-    @Column(
-            name = "gameMasterID",
-            nullable = false
-    )
+    @Column(name = "gameMasterID",
+            nullable = false)
     private Long gameMasterID;
 
-    @Column(
-            name = "mapID"
-    )
+    @Column(name = "mapID")
     // TODO mapID will be the foreign key for maps database, where details about the map are stored
     private Long mapID;
 
-    @Column(
-            name = "roomCode",
-            unique = true,
-            nullable = false
-    )
-    // Game code exchanged by players in order to join the play
-    private String roomCode;
-
-    @Column(
-            name = "turnTime"
-    )
+    @Column(name = "roundTime")
     // Duration of each turn in seconds (for a player to make move)
-    private Integer turnTime;
+    private Integer roundTime;
 
-    @Column(
-            name = "limitOfPlayers"
-    )
+    @Column(name = "limitOfPlayers")
     // Maximal number of players (apart from Game Master)
     private Integer limitOfPlayers;
+
+    @ElementCollection
+    @Column(name = "userList")
+    private List<Long> userList;
+
+//    @ElementCollection
+//    @Column(name = "gamesList")
+//    private List<Long> gamesList; // kiedy bedzie wiecej gier w 1 pokoju
+
+    public GameRoom(Long mapId, int playersLimit, int roundTime, Long gameMasterId) {
+        this.mapID = mapId;
+        this.limitOfPlayers = playersLimit;
+        this.roundTime = roundTime;
+        this.gameMasterID = gameMasterId;
+    }
+
+    public GameRoom() {
```

Comment:
Czy pusty konstruktor jest tutaj celowo? Może zapomniałaś usunąć?

Reply:
Właśnie to jest chyba tak, że on musi tam być, jakaś własność chyba tego, że to jest z `@Entity`